parsec/core/gui/bootstrap_organization_widget.py

```diff
@@ -30,25 +31,16 @@
 from parsec.core.gui.ui.bootstrap_organization_widget import Ui_BootstrapOrganizationWidget
 
 
-_translate = QCoreApplication.translate
-
-
 STATUS_TO_ERRMSG = {
-    "invalid-url": _translate(
-        "BootstrapOrganizationWidget", "This organization does not exist (is the URL correct ?)."
-    ),
-    "user-exists": _translate("BootstrapOrganizationWidget", "This user already exists."),
-    "password-mismatch": _translate("BootstrapOrganizationWidget", "Passwords don't match."),
-    "password-size": _translate(
-        "BootstrapOrganizationWidget", "Password must be at least 8 caracters long."
-    ),
-    "bad-url": _translate("BootstrapOrganizationWidget", "URL or device is invalid."),
-    "bad-device_name": _translate("BootstrapOrganizationWidget", "URL or device is invalid."),
-    "bad-user_id": _translate("BootstrapOrganizationWidget", "URL or device is invalid."),
+    "invalid-url": _("This organization does not exist (is the URL correct ?)."),
```

Comment:
À quel moment la traduction est réalisée ?
Si l'objet retourné par `_("Don't foo the bar")` est lazy ça marche de faire l'appel à cet endroit, par contre si la conversion est faite directement alors on va se retrouver avec le text traduit avec les paramètres de langues configurés au moment du chargement du module (ce qui risque bien de ne pas être les bon, et va complexifier les tests)

Au final vu qu'on a beaucoup simplifié niveau verbosité, je pense qu'on peut remettre ces strings dans le corp de la fonction qui les utilises (avec des ifs à la place du lookup du dico)

Reply:
J'ai bougé les messages dans le corps de la fonction qui les utilise.